tests/integration/states/test_user.py

```diff
@@ -34,33 +34,30 @@
     NOGROUPGID = 'nogroup'
 
 
+@destructiveTest
+@skipIf(os.geteuid() != 0, 'you must be root to run this test')
```

Comment:
No need to change but `tests.support.helpers` has a decorator that skips test if not root in *nix or ADMINISTRATOR in windows.